test/parallel/test-http-get-pipeline-problem.js

```diff
@@ -32,7 +33,7 @@ http.globalAgent.maxSockets = 1;
 
 common.refreshTmpDir();
 
-const image = fs.readFileSync(`${common.fixturesDir}/person.jpg`);
+const image = fs.readFileSync(fixtures.path('/person.jpg'));
```

Comment:
This could be using method `fixtures.readSync` instead of `fs.readFileSync` and `fixtures.path`.